packages/tools/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -62,7 +66,48 @@ export default async args => {
                 }),
                 ...(i === 0 ? getClientComponentPlugins() : []),
                 json(),
-                collectDependencies({ outputDir: outDir, onlyNative: true, widgetName }),
+                collectDependencies({
+                    outputDir: outDir,
+                    onlyNative: true,
+                    widgetName,
+                    ...(production && i === 0
+                        ? {
+                              licenseOptions: {
+                                  thirdParty: {
+                                      output: [
+                                          {
+                                              file: join(outDir, "dependencies.txt")
+                                          },
+                                          {
+                                              file: join(outDir, "dependencies.json"),
+                                              template: dependencies =>
+                                                  JSON.stringify(
```

Comment:
Maybe extract a function, to reduce the nesting a bit

Reply:
done